CoreTableView/test/unit/src/au/gov/asd/tac/constellation/views/tableview2/ShowInTableViewContextMenuProviderNGTest.java

```diff
@@ -0,0 +1,139 @@
+/*
+ * Copyright 2010-2021 Australian Signals Directorate
+ * 
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package au.gov.asd.tac.constellation.views.tableview2;
+
+import au.gov.asd.tac.constellation.graph.GraphElementType;
+import java.util.List;
+import javax.swing.SwingUtilities;
+import static org.mockito.ArgumentMatchers.any;
+import org.mockito.MockedStatic;
+import org.mockito.Mockito;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+import org.openide.windows.WindowManager;
+import static org.testng.AssertJUnit.assertEquals;
+import static org.testng.AssertJUnit.assertTrue;
+import org.testng.annotations.Test;
+
+/**
+ *
+ * @author formalhaunt
+ */
+public class ShowInTableViewContextMenuProviderNGTest {
+ 
+    @Test
+    public void getMenuPath() {
+        assertTrue(new ShowInTableViewContextMenuProvider()
+                .getMenuPath(GraphElementType.META).isEmpty());
+    }
+    
+    @Test
+    public void getItems() {
+        final ShowInTableViewContextMenuProvider provider = new ShowInTableViewContextMenuProvider();
+        assertEquals(provider.getItems(null, GraphElementType.VERTEX, -1), List.of("Show in New Table View"));
+        assertEquals(provider.getItems(null, GraphElementType.TRANSACTION, -1), List.of("Show in New Table View"));
+        assertTrue(provider.getItems(null, GraphElementType.META, -1).isEmpty());
+    }
+    
+    @Test
+    public void selectItemTopComponentNotOpen() {
```

Comment:
I don't think I can see any assertions in this unit test. I might've missed something though. If not, could be nice to include some if possible

Reply:
Its the verify in the thenAnswer section. The test basically just ensures that those methods are called. You will see that they change between the different tests.

They are in here rather than at the end of the test due to the multi threaded nature of the code.